example/src/TableExample.js

```diff
@@ -0,0 +1,207 @@
+            </TableCell>
+          </TableRow>
+          <TableRow style={styles2.row}>
+            <TableCell style={styles2.cell}>Data 1</TableCell>
+            <TableCell style={styles2.cell}>Data 2</TableCell>
+            <TableCell style={styles2.cell} numeric>
+              30.00
+            </TableCell>
+            <TableCell style={styles2.cell} numeric>
+              3000
+            </TableCell>
+          </TableRow>
+          <TableRow style={styles2.row}>
+            <TableCell style={styles2.cell}>Data 1</TableCell>
+            <TableCell style={styles2.cell}>Data 2</TableCell>
+            <TableCell style={styles2.cell} numeric>
+              30.00
+            </TableCell>
+            <TableCell style={styles2.cell} numeric>
+              3000
+            </TableCell>
+          </TableRow>
+          <TableRow style={styles2.row}>
+            <TableCell style={styles2.cell}>Data 1</TableCell>
+            <TableCell style={styles2.cell}>Data 2</TableCell>
+            <TableCell style={styles2.cell} numeric>
+              30.00
+            </TableCell>
+            <TableCell style={styles2.cell} numeric>
+              3000
+            </TableCell>
+          </TableRow>
+          <TableRow style={styles2.row}>
+            <TableCell style={styles2.cell}>Data 1</TableCell>
+            <TableCell style={styles2.cell}>Data 2</TableCell>
+            <TableCell style={styles2.cell} numeric>
+              30.00
+            </TableCell>
+            <TableCell style={styles2.cell} numeric>
+              3000
+            </TableCell>
+          </TableRow>
+        </Table>
+      </Section>
+    </Container>
+  );
+}
+
+const styles = StyleSheet.create({
+  table: {
+    width: "100%",
+  },
+  header: {
+    paddingLeft: 16,
+    paddingRight: 16,
+    borderBottomWidth: 1,
+    borderStyle: "solid",
+    backgroundColor: "#EFEFEF",
+  },
+  title: {
+    fontSize: 14,
+    fontWeight: "500",
+    paddingTop: 16,
+    paddingBottom: 16,
+    paddingLeft: 8,
+    paddingRight: 8,
+  },
+  row: {
+    paddingLeft: 16,
+    paddingRight: 16,
+    borderBottomWidth: 1,
+    borderStyle: "solid",
+  },
+  cell: {
+    paddingTop: 16,
+    paddingBottom: 16,
+    paddingLeft: 8,
+    paddingRight: 8,
+  },
+});
+
+const styles2 = StyleSheet.create({
+  table: {
+    width: "100%",
+  },
+  header: {
+    paddingLeft: 16,
+    paddingRight: 16,
+    borderBottomWidth: 1,
+    borderStyle: "solid",
+    backgroundColor: "#EFEFEF",
+    borderBottomColor: "#ff0000",
+  },
+  title: {
+    fontSize: 14,
+    fontWeight: "500",
+    paddingTop: 16,
+    paddingBottom: 16,
+    paddingLeft: 8,
+    paddingRight: 8,
+    color: "#0000FF",
+  },
+  row: {
+    paddingLeft: 16,
+    paddingRight: 16,
+    borderBottomWidth: 1,
+    borderStyle: "solid",
+  },
+  cell: {
+    paddingTop: 16,
+    paddingBottom: 16,
+    paddingLeft: 8,
+    paddingRight: 8,
+  },
+});
+
+export default withTheme(TableExample);
```

Comment:
Lets remove theme and theme objects from these and instead make sure we can just style them with props, and set reasonable defaults on those props.

Reply:
removed